.github/workflows/test-coverage.yml

```
@@ -15,28 +15,35 @@
       contents: read
       pull-requests: read
       checks: write
+      security-events: write
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)